packages/@react-spectrum/progress/docs/ProgressBar.mdx

```diff
@@ -0,0 +1,121 @@
+```
+
+Alternatively, a different scale can be used by setting the `minValue` and `maxValue`.
+
+```tsx example
+<ProgressBar label="Loading…" minValue={50} maxValue={150} value={100} />
+```
+
+Values are formatted by default as a percentage, but can modified via `formatOptions` for different formats.
+The `formatOptions` are displayed via the [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) applied for the current locale.
+
+```tsx example
+<ProgressBar label="Loading…" formatOptions={{style: 'currency', currency: 'JPY'}} value={60} />
+```
+
+### Accessibility
+
+Progress Bars are made accessible as per the ["progressbar" role in WAI-ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#progressbar).
+Depending on the visualization being used (ie depending on the `showValueLabel` prop), a `label`, `aria-label`, or `aria-labelledby` attribute are required.
+
+```tsx example
+<div>
+  <ProgressBar aria-labelledby="pc-label" showValueLabel={false} />
+  <label id="pc-label" style={{display: 'block'}}>Loading…</label>
+</div>
+```
+
+If using the over background variant, ensure the background offers enough contrast for the loader to be legible and meets color contrast guidelines.
+
+### Internationalization
+
+To internationalize a Progress Bar, a localized string should be passed to the `label` prop, `aria-label` prop, or value of associated `aria-labelledby` element.
+
+For RTL (right-to-left) languages, the layout of the bar loader is mirrored for both determinate and indeterminate options.
+The placement of the percent sign differs depending on the locale.
+
+## Props
+
+<PropTable component={docs.exports.ProgressBar} links={docs.links} />
+
+## Visual Options
+
+### Over background
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Over-background)
+
+When a loader needs to be placed on top of a colored background, use the over background loader.
+
+```tsx example
+<div style={{background: 'rgba(0,0,0,0.4)', padding: '10px'}}>
+  <ProgressBar label="Loading…" variant="overBackground" />
+</div>
+```
+
+### Indeterminate
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Determinate-or-indeterminate)
+
+By default, loaders are determinate. Use a determinate loader when progress can be calculated against a specific goal.
+
+```tsx example
+<ProgressBar label="Loading…" value={50} />
+```
+
+Use an indeterminate loader when progress is happening but the time or effort to completion can’t be determined.
+
+```tsx example
+<ProgressBar label="Loading…" isIndeterminate />
+```
+
+### Labels
+[View guidelines](https://spectrum.adobe.com/page/bar-loader/#Label-or-no-label)
+
+Value labels are shown by default on top, but they can also be moved to the side or hidden as needed.
+
+```tsx example
+<p><ProgressBar label="Loading…" /></p>
+<p><ProgressBar label="Loading…" labelPosition="side" /></p>
+<p><ProgressBar label="Loading…" showValueLabel={false} /></p>
```

Comment:
Would be good to include a value for all examples. They look strange if the value is 0 IMO.